lib/reverse_adoc/converters/blockquote.rb

```diff
@@ -3,13 +3,13 @@ module Converters
     class Blockquote < Base
       def to_coradoc(node, state = {})
         id = node['id']
-        anchor = id ? "[[#{id}]]\n" : ""
         cite = node['cite']
-        attrs = cite ? "[quote, #{cite}]\n" : ""
+        attributes = cite.nil? ? nil :  Coradoc::Document::AttributeList.new("quote", cite)
         content = treat_children(node, state).strip
         content = ReverseAdoc.cleaner.remove_newlines(content)
-        Coradoc::Document::Block.new(nil, lines: content.lines, type: :quote, attributes: attrs)
+        Coradoc::Document::Block::Quote.new(nil, lines: content, attributes: attributes)
```

Comment:
What’s the first nil argument? It looks weird.

Reply:
It's title, previous reverse_adoc code was inconsistent with regards to blocks, I will look into how exactly are different types of asciidoc blocks converted into html

Reply:
So there are actually 3 types of HTML source:

1. Normal HTML
2. Source generated by Metanorma
  * Source: https://github.com/metanorma/mn-samples-iso
  * HTML: https://metanorma.github.io/mn-samples-iso/
3. Source generated by Word
  * (@opoudjis what are the examples?)